packages/sdk/src/types/runtime/collider.ts

```diff
@@ -3,54 +3,64 @@
  * Licensed under the MIT License.
  */
 
-import { Actor, ColliderParams, CollisionLayer } from '.';
+import { Actor, ColliderGeometry } from '.';
 
 /**
  * Describes the properties of a collider.
  */
 export interface ColliderLike {
     enabled: boolean;
     isTrigger: boolean;
-    collisionLayer: CollisionLayer;
-    colliderParams: ColliderParams;
+    // collisionLayer: CollisionLayer;
+    colliderGeometry: ColliderGeometry;
 }
 
 /**
  * A collider represents the abstraction of a physics collider object on the host.
  */
 export class Collider implements ColliderLike {
-    public enabled: Readonly<boolean>;
-    public isTrigger: boolean;
-    public collisionLayer: CollisionLayer;
+    public enabled = true;
+    public isTrigger = false;
+    // public collisionLayer = CollisionLayer.Object;
 
     // Readonly params that are not patchable or observable.
     // tslint:disable-next-line:variable-name
-    private _colliderParams: Readonly<ColliderParams>;
+    private _colliderGeometry: Readonly<ColliderGeometry>;
 
-    public get colliderParams() { return this._colliderParams; }
+    /**
+     * The collider geometry that the collider was initialized with.  These are a
+     * readonly structure and are not able to be updated after creation.
+     */
+    public get colliderGeometry() { return this._colliderGeometry; }
 
     /**
      * Creates a new Collider instance.
      * @param $owner The owning actor instance. Field name is prefixed with a dollar sign so that it is ignored by
+     * @param initFrom The collider like to use to init from.
      * the actor patch detection system.
      */
-    constructor(private $owner: Actor) { }
-
-    public copy(from: Partial<ColliderLike>): this {
-        if (!from) return this;
-        if (from.enabled !== undefined) this.enabled = from.enabled;
-        if (from.isTrigger !== undefined) this.isTrigger = from.isTrigger;
-        if (from.collisionLayer !== undefined) this.collisionLayer = from.collisionLayer;
-        if (from.colliderParams !== undefined) this._colliderParams = from.colliderParams;
-        return this;
+    constructor(private $owner: Actor, initFrom: Partial<ColliderLike>) {

```

Comment:
>constructor [](start = 4, length = 11)

Do you want to mark the constructor @hidden? I think we want it excluded from documentation.

Reply:
Yep. Added.